docs/Makefile

```diff
@@ -1,24 +1,27 @@
-.PHONY: docs docs-no-pull docs-test docs-local-static
+.PHONY: pull docs docs-quick docs-no-pull docs-test docs-local-static
```

Comment:
What are your thoughts on naming this (alias) `docs-mpv` and the target is `docs-minus-previous-versions`?

Reply:
I am not sure based on looking at the `docs-quick` what is actually happening because `quick` seems ambiguous to me. I could be doing many things to make the build quicker.

Reply:
we are using the term `quick` here as that is what the environment is called: https://github.com/grafana/website/blob/master/Makefile#L151